sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs

```diff
@@ -1535,7 +1535,7 @@ public virtual async Task<Operation<BinaryData>> DeleteUpdateAsync(WaitUntil wai
         /// </code>
         /// 
         /// </remarks>
-        public virtual Operation<BinaryData> DeleteUpdate(WaitUntil waitUntil, string provider, string name, string version, RequestContext context = null)
+        public virtual Operation DeleteUpdate(WaitUntil waitUntil, string provider, string name, string version, RequestContext context = null)
```

Comment:
@mikekistler, can you confirm that there is no value returned from the `DeleteUpdate` LRO?

Reply:
I cannot confirm what is returned from this operation.  I hoped to determine this empirically by setting up a test environment but it proved too complicated.

Reply:
@dpokluda, could you confirm that there is no value returned from the DeleteUpdate LRO?